### DISCUSSION TITLE
video07

### DISCUSSION THREAD

**Discussion by mcharlie9** [jacekschae/learn-pedestal-course-files#3] [Feedback]
1) The portion from beginning to [0:07] can be edited out.

2) [1:40-1:50] can be edited out, but the editing may take a bit more work. The line "is required by another library" from [1:40-1:42] should be kept while the screen should remain at [1:40], after the line finishes, continuing at [1:50] should make for a smooth transition. Just a suggestion, please edit in whatever way you see fit.

**Comment by jacekschae** [comment]
Thanks being so detailed. Just uploaded with corrections.
